index.js

```diff
@@ -125,18 +125,44 @@ function Linux(instance, end) {
       }
       command.push(instance.command);
       command = command.join(' ');
-      Node.child.exec(command, { maxBuffer: MAX_BUFFER },
+      var cp = Node.child.exec(command, { maxBuffer: MAX_BUFFER },
```

Comment:
It would be better to specify the encoding explicitly as "utf-8" instead of assuming it as the default.